.gitignore

```diff
@@ -21,4 +21,4 @@ e2e-tests
 concurrent_workspace_script_cache.json
 
 testDir
-.amplify
+/.amplify
```

Comment:
Update change from #984 to only ignore `.amplify` at the root because we want to check in `packages/integration-tests/src/data-storage-auth-with-triggers-ts/.amplify`